src/Intervals.jl

```diff
@@ -2,13 +2,11 @@ __precompile__()
 
 module Intervals
 
-using Compat
-using Compat.Dates
-using Compat.Printf
+using Dates
+using Printf
 using TimeZones
 
-using Compat.Dates: value, coarserperiod
-using Compat: AbstractDateTime
+using Dates: value, coarserperiod, AbstractDateTime
```

Comment:
I think our preferred order has types first:
```suggestion
using Dates: AbstractDateTime, coarserperiod, value
```